server.js

```diff
@@ -151,6 +154,41 @@ server.get('/logout', function(req, res){
   res.redirect('/')
 })
 
+server.get('/instagram', function(req, res){
+  if(req.user){
+    if(req.query.m == 123){
```

Comment:
To było dla testów, spokojnie możesz to usunąć ;) tam tylko trzeba sprawdzać czy istnieje code w adresie ;)